libpod/pod_api.go

```diff
@@ -597,6 +598,11 @@ func (p *Pod) Inspect() (*define.InspectPodData, error) {
 		infraConfig.CPUSetCPUs = p.ResourceLim().CPU.Cpus
 		infraConfig.PidNS = p.PidMode()
 		infraConfig.UserNS = p.UserNSMode()
+		namedVolumes, mounts := infra.sortUserVolumes(infra.Config().Spec)
```

Comment:
This is wrong - named volumes are a separate field in the infra container's config, they are not in the spec

Reply:
see comment below